### PR TITLE
Fix translations for unitssss

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #

# 👩🏻‍💻 What does this PR do? 
From 
Screenshot 2023-08-18 at 4 04 50 PM
to
Screenshot 2023-08-18 at 4 04 16 PM
Probably only bugging out for me though 😭 

# 🧪 How has/should this change been tested? 
- [ ] Add an item that doesn't have a unit name to an invoice line and shouldn't see the extra 's'

## 📃 Documentation

_No user facing changes_

or

- [ ] Functional change 1
- [ ] Functional change 2

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/distribution.json [hunk 1]
```diff
@@ -60,10 +60,6 @@
   "label.requested-quantity": "Requested quantity",
   "label.select-rows-to-allocate-them": "Select rows to allocate them",
   "label.supply-quantity": "Quantity to supply",
-  "label.unit_one": "{{unit}}",
```
Comment (roxy-dao) [inline client/packages/common/src/intl/locales/en/distribution.json:63]: couldn't find this label.unit_one in the es translation file so have deleted from all

**Review by mark-prins — CHANGES_REQUESTED** [review]
`catalogue.json` has `label.units` and this can be removed now
`tet/distribution.json` has 
```
  "label.units_one": "{{Unidade}}",
  "label.units_other": "{{Unidade}}s",
  "label.units-in-pack-size-of_one": "{{Unidade}} iha pakote husi",
  "label.units-in-pack-size-of_other": "{{Unidade}}s iha pakote husi",
```

all of which can go too!

The OutboundLineEditForm and the PrescriptionLineEditForm are both using `'label.unit'` with a `count` prop though. The 'label.unit_one' and 'label.unit_other' were removed from distribution and not added to common.. but they will need to be. and yet most usages are wanting the translation 'Unit'.

I would suggest:
- changing `label.units-in-pack-size-of_one` to `label.in-packs-of` with a translation of `in packs of`
- removing `label.units-in-pack-size-of_other`
- adding `"label.unit-plural_one": "{{unit}}` to common.json
- adding `"label.unit-plural_other": "{{unit}}s` to common.json
- changing the two usages of `label.unit` mentioned above
- changing the usage of `label.units-in-pack-size-of` to `t('label.unit-plural', { count: xxx}) t('label.in-packs-of')`

does that make sense?
side note: I'm still unable to reproduce the original issue! using develop branch, it's fine for me 🤷

**Comment by roxy-dao** [comment]
> does that make sense? side note: I'm still unable to reproduce the original issue! using develop branch, it's fine for me 🤷

So odd... been happening to me since plural was implemented 😭

File: client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx [hunk 2]
```diff
@@ -179,11 +185,11 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
...
                   <InputLabel sx={{ fontSize: '12px' }}>
                     {packSizeController.selected?.value === -1
-                      ? t('label.units-in-pack-size-of', {
+                      ? `${t('label.unit-plural', {
                           unit,
                           count: quantity,
-                        })
-                      : t('label.packs-of', { count: quantity })}
+                        })} ${t('label.in-packs-of')}`
+                      : t('label.in-packs-of', { count: quantity })}
```
Comment (mark-prins) [inline client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx:192]: this block (lines 188-192) should be:

```
                      ? `${t('label.unit-plural', {
                          unit,
                          count: issueQuantity,
                        })} ${t('label.in-packs-of')}`
                      : t('label.in-packs-of')}
```

the quantity should be `issueQuantity` and `label.in-packs-of` has no params now

File: client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx [hunk 3]
```diff
@@ -76,7 +76,8 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
     setShowAllocationWarning(false);
   };
 
-  const unit = item?.unitName ?? t('label.units', { count: 1 });
+  const unit = item?.unitName ?? t('label.unit');
+  console.log(item?.unitName);
```
Comment (mark-prins) [inline client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx:80]: can remove?

File: client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx [hunk 4]
```diff
@@ -189,10 +189,10 @@ export const PrescriptionLineEditForm: React.FC<
                 >
                   <InputLabel sx={{ fontSize: '12px' }}>
                     {packSizeController.selected?.value === -1
-                      ? t('label.units-in-pack-size-of', {
+                      ? `${t('label.unit-plural', {
```
Comment (mark-prins) [inline client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx:192]: the `count` param here should be `issueQuantity`

Comment (mark-prins) [reply client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx:192]: and line 196 should be 

```
t('label.packs-of')}
```

(remove the `, { count: quantity }`) 🙏

**Review by mark-prins — CHANGES_REQUESTED** [review]
thanks Roxy - a couple more changes please; partly tidy up, and partly the plurals not working on the unit translations

**Review by mark-prins — APPROVED** [review]
🙏  thanks again Roxy!
Found a couple more bugs - this time I've just committed directly in 3e91aafd5 rather than go back and forth again.
hopefully that's the last of them!

**Comment by roxy-dao** [comment]
Thanks Mark!

**Pull request merged**